content/documentation/pages/6-recipes/4-kafka/1-ext-kafka-cluster-cf.md

```diff
@@ -83,27 +93,55 @@ With that flexibility, every stream application deployed through SCDF will autom
 ```yaml
 ---
 applications:
-- name: scdf-server
-  host: scdf-server
-  memory: 2G
-  disk_quota: 2G
-  timeout: 180
-  instances: 1
-  path: spring-cloud-dataflow-server-2.1.0.RELEASE.jar
+  - name: scdf-server
+    host: scdf-server
+    memory: 2G
+    disk_quota: 2G
+    timeout: 180
+    instances: 1
+    path: spring-cloud-dataflow-server-2.1.0.RELEASE.jar
```

Comment:
Same as above. It'd be nice to pull the latest version from the `variables.json`